test_data/rules/use_decorated_box.dart

```diff
@@ -20,7 +20,7 @@ Widget containerWithKey() {
 
 Widget containerWithDecoration() {
   return Container( // LINT
-    decoration: BoxDecoration(),
+    decoration: Decoration(),
```

Comment:
I believe that these changes (from `BoxDecoration` to `Decoration`) also need to be reverted. It's a false positive to lint here because this code can't use `DecoratedBox`.